packages/jest-editor-support/src/__tests__/fixtures/snapshots/__snapshots__/react-native.example.snap

```diff
@@ -0,0 +1,105 @@
+exports[`test renders correctly 1`] = `
```

Comment:
Both the two example tests and two example snapshots are rather long. Can we condense them a little bit to be smaller in size? That'll make it easier to maintain them and it gets rid of all the things in the code that we don't need.